app/components/school_information/details/main_info_component.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-m govuk-!-margin-bottom-2">
+      School information
+    </h1>
+
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          URN
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= @school.urn %>
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Local authority
+        </dt>
+        <dd class="govuk-summary-list__value">
+          Essex
+        </dd>
+      </div>
+
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          School contact
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <a href="#"><%= @school.primary_contact_email %></a>
```

Comment:
We shouldn't be showing this anywhere on the frontend. We will maybe show the induction coordinator's email, but never the GIAS primary contact.

Reply:
From the designs I had assumed that it would be primary contact email - thanks for pointing it out

Reply:
Added a TODO and hardcoded the email